libs/components/core/src/lib/modules/overlay/overlay.component.ts

```diff
@@ -281,4 +289,28 @@ export class SkyOverlayComponent implements OnInit, OnDestroy {
       this.#routerSubscription = undefined;
     }
   }
+
+  #addScrollListener(): void {
+    if (
+      this.#doc.defaultView?.visualViewport &&
+      this.#context.config.position === 'fixed'
+    ) {
+      // Safari on iOS allows fixed position elements to scroll with the page.
+      fromEvent(this.#doc.defaultView.visualViewport, 'scroll')
+        .pipe(takeUntil(this.#ngUnsubscribe))
+        .subscribe(() => {
+          /* istanbul ignore else */
+          if (
+            this.#doc.defaultView?.visualViewport &&
+            this.overlayRef?.nativeElement
+          ) {
+            this.#renderer.setStyle(
+              this.overlayRef?.nativeElement,
+              'top',
+              `${this.#doc.defaultView.visualViewport.offsetTop}px`
+            );
+          }
+        });
+    }
+  }
```

Comment:
This is 3/3 of the fix for mobile Safari, which was the original issue where we tried using absolute position.